tools/fv_diagnostics.F90

```diff
@@ -595,8 +605,10 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
 !-------------------
 ! Precipitation from GFDL MP
 !-------------------
-       id_prec = register_diag_field ( trim(field), 'prec', axes(1:2), Time,           &
+       id_pret = register_diag_field ( trim(field), 'pret', axes(1:2), Time,           &
```

Comment:
This is a change that needs to be mentioned in the release notes, since it changes the output of an important diagnostic.

Reply:
Would this summarize the change suficiently if I were to add it to the release notes: 
"total precipitation diag field has changed from ```prec``` to ```pret```"

Reply:
Yes. Thank you.

Reply:
I made this change so that prec represents convective precipitation. pret becomes the total precipitation. Sorry for my oversight.